api/Event.json

```
@@ -284,10 +284,10 @@
               "notes": "Starting with Chrome 58 and Opera 45, setting this property to false does nothing, as per <a href='https://github.com/whatwg/dom/issues/211'>spec discussion</a>."
             },
             "safari": {
-              "version_added": "≤4"
+              "version_added": "3"
```

I don't see a source for why `cancelBubble` should be in Safari 3. I think it should be 1 based on https://github.com/WebKit/WebKit/blob/b923c2f4eb7804aae65d67b4c3061d6435acb10d/WebCore/khtml/ecma/kjs_events.cpp#L189

```suggestion
              "version_added": "1"
```